util.go

```diff
@@ -61,7 +61,7 @@ func (twilio *Twilio) GenerateSignature(url string, form url.Values) ([]byte, er
 // header is an error.
 func (twilio *Twilio) CheckRequestSignature(r *http.Request, baseURL string) (bool, error) {
 	if r.Method != "POST" {
-		return false, errors.New("Checking signatures on non-POST requests is not implemented")
+		return false, errors.New("checking signatures on non-POST requests is not implemented")
```

Comment:
Linter told me to change this to lowercase.

Reply:
Yes - that's right. All lowercase and no punctuation for error messages.